rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMPage.java

```diff
@@ -500,6 +506,18 @@ public void onClick(ClickEvent event) {
         return cloudAutomationButton;
     }
 
+    private ToolStripButton getNotificationPortalLinkButton() {
+        ToolStripButton notificationPortalButton = new ToolStripButton("Notifications");
+
+        notificationPortalButton.addClickHandler(new ClickHandler() {
+            public void onClick(ClickEvent event) {
+                Window.open("/notification-service", "", "");
```

Comment:
Using _/notification-service_ assumes that all ProActive services are deployed on the same host. We should probably start thinking in the future about a configuration file that defines the location of the services.